ACounter/Program.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ACounter
+{
+  class Program
+  {
+    static void Main(string[] args)
+    {
+      string str = Console.ReadLine();
```

Comment:
I would also add some notes for the user:

Console.Writline("Input a text, please"); - for instance